app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,80 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
+
+  before_action :not_found, unless: :endpoint_set?
+  before_action :send_pdf, if: :pdf?
+  before_action :fetch_nav, only: :show
+  before_action :proxy_content, only: :show
```

Comment:
I don't like the fact that these are all before actions – it makes the code needlessly hard to follow in the action.